bin/packages/lib/php-generator.mjs

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import { readFile, writeFile, mkdir } from 'fs/promises';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __dirname = path.dirname( fileURLToPath( import.meta.url ) );
+
+/**
+ * Get PHP replacements from root package.json.
+ *
+ * @param {string} rootDir Root directory path.
+ * @return {Promise<Object>} Replacements object with {{PREFIX}}, {{VERSION}}, {{VERSION_CONSTANT}}.
```

Comment:
I've typically seen `Object` be discouraged in favor of `object` or `Record<...>` types

https://typescript-eslint.io/rules/no-wrapper-object-types/